patches/api/0304-Missing-Entity-Behavior-API.patch

```diff
@@ -1197,7 +1198,32 @@ index a4411daca0d8a770b0d15e08847b82df8a3ec27f..50464aa8c38cd47fc049ee022e25cb49
 +     * @return number of ticks or -1 if the fuse isn't primed
 +     */
 +    int getFuseTicks();
-+    // Paper end
++
++    /**
++     * Checks whether the fuse is primed
++     *
++     * @return whether the fuse is primed
++     */
++    boolean isPrimed();
```

Comment:
Method should be isIgnited() as it is on a Creeper. 

(Another PR should add this method to PrimedTNT)

Reply:
And sure, I'll make a PR adding the method to PrimedTNT later